content/zh/docs/concepts/storage/projected-volumes.md

```diff
@@ -49,10 +49,11 @@ Currently, the following types of volume sources can be projected:
 
 <!--
 All sources are required to be in the same namespace as the Pod. For more details,
-see the [all-in-one volume design document](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/node/all-in-one-volume.md).
+see the [all-in-one volume](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/node/all-in-one-volume.md).
```

Comment:
delete this line

Reply:
How could I missed that!

Thanks very much, fixed. 👍